coalib/results/result_actions/ResultAction.py

```diff
@@ -61,7 +62,16 @@ def apply_from_section(self,
         :return:                   The modified file_diff_dict.
         """
         params = self.get_metadata().create_params_from_section(section)
-        return self.apply(result, original_file_dict, file_diff_dict, **params)
+        if 'context_line_nr' in kwargs:
+            context_line_nr = kwargs.pop('context_line_nr')
+            return self.apply(result,
+                              original_file_dict,
+                              file_diff_dict,
+                              context_line_nr, **params)
+        else:
+            return self.apply(result,
+                              original_file_dict,
+                              file_diff_dict, **params)
```

Comment:
I don't think adding this here makes sense, this information is only valid for a given diff and should be generated on application of the showpatchaction or so, this has nothing to do in the class scope